editor/components/CrashRecommendationCard.tsx

```diff
@@ -204,7 +204,7 @@ export default function CrashRecommendationCard({
             )}
             {isEditing && isLoadingPartners && <Spinner size="sm" />}
             {isEditing && partners && (
-              <div style={{ height: "200px", overflowY: "scroll" }}>
+              <div style={{ height: "200px", overflowY: "auto" }}>
```

Comment:
`auto` is preferable because `scroll` causes the scrollbar to be always visible, even when content does not overflow.